ftw/zipextract/File.py

```diff
@@ -0,0 +1,27 @@
+#from opengever.document.document import IDocumentSchema
+from zope.component import adapts
+from zope.interface import implements
+from interfaces import IFile
+
+
+class _FileBase():
```

Comment:
I would not prefix classes with an underscore. You could add a docstring indicating that this class is meant as abstract base class instead.
I'd probably also declare not implemented methods like this:
```python
def get_blob(self):
    raise NotImplementedError()
```